### PR TITLE
Update user guide on the usage of `Availability`

### LINKED ISSUE #81: Update features that uses the `Availability` attribute in user guide
### Segments to edit:
- [ ] Feature: `add`
- [ ] Feature: `edit`
- [ ] Command summary: `add` and `edit`

### PR DESCRIPTION
**Pull request by tsulim**
Closes #81 
### Segments to edit:
- Feature: `add`
- Feature: `edit`
- Command summary: `add` and `edit`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -96,14 +96,17 @@ Format: `list`
...
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [v/VENUE] [m/MODULE] [t/TAG]…​ [a/AVAILABILITY]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `t/` without
     specifying any tags after it.
+* When editing availabilities, the existing availabilites of the person will be removed i.e adding of availabilities is not cumulative.
+* You can remove all the person’s availabilities by typing `a/` without
```
Comment (Pluiexo) [inline docs/UserGuide.md:108]: Would like to link the same issue in #80 ,  would the empty list indicate availability for all days of the week?

**Review by Pluiexo — COMMENTED** [review]
LGTM! Just the same issue questioned in the implementation PR

File: docs/UserGuide.md [hunk 2]
```diff
@@ -189,10 +192,10 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL v/VENUE m/MODULE [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com v/123, Clementi Rd, 1234665 m/CS2103 t/friend t/colleague`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL v/VENUE m/MODULE [t/TAG]…​ [a/AVAILABILITY]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com v/123, Clementi Rd, 1234665 m/CS2103 t/friend t/colleague`
```
Comment (JerryWang0000) [inline docs/UserGuide.md]: If there is an example of more than 0 availablity it's better

Comment (whitesnowx) [reply docs/UserGuide.md]:
```suggestion
**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL v/VENUE m/MODULE [t/TAG]…​ [a/AVAILABILITY]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com v/123, Clementi Rd, 1234665 m/CS2103 t/friend t/colleague a/monday`
```

**Review by whitesnowx — COMMENTED** [review]
LOOK GOOD TO ME 🤸‍♂️ 

/APPROVE

**Review by whitesnowx — APPROVED** [review]

**Pull request merged**